tutorials/en/vectara-advanced-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-or-customer-support-use-case.mdx

```diff
@@ -0,0 +1,671 @@
+
+#### **1. A Glimpse into the Interface:**
+
+Our application offers a clean and intuitive interface to users. The landing page is straightforward, featuring a sidebar dedicated to Hackathon resources and two expandable sections for document indexing and corpus searching.
+
+<div align="center">
+
+![Interface Glimpse](https://iili.io/Jftsd1j.md.png)
+
+</div>
+
+#### **2. Document Indexing: Your Gateway to Knowledge**
+
+Within the interface, users have the capability to upload documents directly to the Vectara corpus. Be it text, a spreadsheet, or a presentation, our system is adept at handling it. An added document title field aids users in giving a descriptive name, facilitating better organization and retrieval.
+
+<div align="center">
+
+![Document Indexing](https://iili.io/JftsA2R.md.png)
+
+</div>
+
+#### **3. Corpus Searching: Unleashing the Power of Vectara**
+
+The centerpiece of our application is the corpus searching function. Users input their queries, set the desired number of results, and can even specify the summarization model and desired language. Hitting the "Search" button activates Vectara's advanced search and summarization algorithms, which pull the most pertinent information from the corpus.
+
+<div align="center">
+
+![Corpus Searching](https://iili.io/JftsG4f.md.png)
+
+</div>
+
+#### **4. Seamless Interaction:**
+
+What sets our application apart is the flawless interaction between the user interface and the Vectara backend. Thanks to a well-organized codebase, split into `Indexing` and `Searching` classes, users experience fluid and error-free interactions with Vectara's APIs. Additionally, leveraging environment variables for sensitive data guarantees a setup ready for production.
+
+<div align="center">
+
+![Seamless Interaction](https://iili.io/JftQvdx.md.png)
+
+</div>
+
+
+
+This showcase paints a vivid picture of what our Vectara Retrieval Augmented System is capable of. The synergy between Vectara's robust search capabilities and Streamlit's interactive interface creates a powerful tool that is poised to revolutionize customer support operations. As we step into the future, our application stands ready to serve as a reliable, efficient, and user-friendly solution for information retrieval and customer engagement.
+
+---
+### **5. Additional Learning Materials on Vectara: Amplify Your Understanding** 📚
+
+Embarking on a journey through the realms of Vectara opens up a trove of learning opportunities. While this tutorial provides a solid foundation, diving deeper into Vectara's documentation and resources will equip you with the knowledge to explore new horizons. Here are some avenues to further enrich your understanding:
+
+#### **1. Vectara Documentation:** 📖
```

Comment:
Perhaps highlight that the documenation also has a playground to experiment with the API: https://docs.vectara.com/docs/rest-api/